plugins/modules/vyos_ospfv3.py

```diff
@@ -68,6 +68,15 @@
           import_list:
             description: Name of import-list.
             type: str
+          interface:
+            description: OSPFV3 area interfaces.
```

Comment:
Change description to "Enable OSPVv3 on an interface for this area" (this is more in line with the VyOS documentation and more descriptive of what happens when you add items here.